packages/teuchos/core/src/Teuchos_Exceptions.hpp

```diff
@@ -163,6 +163,24 @@ class DuplicateValidatorIDException : public ExceptionBase {
 
 };
 
+/**
+ * @brief Exception class for non-printable parameter entries,
+ * such as enum class/std::vector and many more
+ * which don't define an operator<<.
+ * Thrown during runtime when trying to print a parameter list
+ * with a non-printable parameter entry.
+ *
+ * \relates ParameterEntry
+ */
+class NonprintableParameterEntryException : public ExceptionBase {
+
+public:
+    NonprintableParameterEntryException(const std::string& what_arg) :
```

Comment:
I forgot this exception is thrown from inside of `Teuchos::any`, and not from within the Teuchos ParameterList software.  It might be better to just rename this `NonpritableTypeException`.

Reply:
done